src/src/components/form/CustomSection.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <th colspan="4" class="sectionHeader">
+    <h3 v-html="field.title" @click="toggle($event)" :class="{'toggle' : field.canToggleSection}"></h3>
```

Comment:
Codacy has a fix for the issue: [Attribute "@click" should go before "v-html".](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
    <h3 @click="toggle($event)" v-html="field.title" :class="{'toggle' : field.canToggleSection}"></h3>
```